### PR TITLE
Prisma Bid Adapter: initial bid adapter release

### PR DESCRIPTION
**Pull request by gchicoye**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ X] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Prisma Bid Adapter added

- test parameters for validating bids
```
{
  bidder: 'prisma',
  params: {
      account: '1067',
      tagId: 'luvxjvgn'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ X] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
`https://github.com/prebid/prebid.github.io/pull/3898`

### CODE REVIEW

**Comment by ChrisHuie** [comment]
Docs pr -> https://github.com/prebid/prebid.github.io/pull/3898

File: modules/prismaBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+          prisma: {
+            'ssp': value['bidder'],
+            'consent': value['consent'],
+            'tagId': value['tagId']
+          },
+          /*
+          meta: {
+            'advertiserDomains': value['adomain']
+          }
+          */
```
Comment (ncolletti) [inline modules/prismaBidAdapter.js:147]: Could this be included in the PR, defaulting to an empty array if `adomain` does not exist on the response

File: modules/prismaBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    const adUnits = [];
+    const test = config.getConfig('debug') ? 1 : 0;
+    let adunitValue = null;
+    let userEids = null;
+    Object.keys(validBidRequests).forEach(key => {
+      adunitValue = validBidRequests[key];
+      const foo = {
+        account: adunitValue.params.account,
+        tagId: adunitValue.params.tagId,
+        videoExt: adunitValue.params.videoExt,
```
Comment (ncolletti) [inline modules/prismaBidAdapter.js:74]: Could this param and it's expected value be documented in your markdown?

**Review by ncolletti — COMMENTED** [review]
This adapter submission is nearly identical to `nexx360` --  Both share the same domain for endpoints, with separate subdomains. Instead of adding a new adapter, has your team considered adding a new alias for `primsa`, when used it would use prisma subdomains for requests.

File: modules/prismaBidAdapter.js (hunk 1 above)
Comment (gchicoye) [reply modules/prismaBidAdapter.js]: Fixed - https://github.com/prebid/Prebid.js/pull/8681/commits/c002549513498e5610be69a98363efd653932366

**Comment by ncolletti** [comment]
Looks good, could your docs PR reflect the removal of bidfloor and bidfloorCurrency, will merge in when updated

**Comment by gchicoye** [comment]
Hi @ncolletti , it's done :) Thanks!

**Review by ncolletti — APPROVED** [review]
lgtm

**Pull request merged**